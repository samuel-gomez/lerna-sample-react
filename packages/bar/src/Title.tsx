import * as React from 'react';

const Title = () => <p>Titless</p>;
export default Title;
