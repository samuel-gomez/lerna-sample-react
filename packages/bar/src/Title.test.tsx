import * as React from 'react';
import { create } from 'react-test-renderer';
import Title from './Title';

describe('<Title>', () => {
  it('should render a Title', () => {
    const component = create(<Title />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
