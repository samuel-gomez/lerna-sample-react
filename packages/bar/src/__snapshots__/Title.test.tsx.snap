// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title> should render a Title 1`] = `
<p>
  Titless
</p>
`;
